src/plugin.js

```diff
@@ -190,6 +196,7 @@ const onPlayerReady = (player, options) => {
         <h2 class="vjs-errors-headline">${this.localize(error.headline)}</h2>
           <div><b>${this.localize('Error Code')}</b>: ${(error.type || error.code)}</div>
           ${details}
+          <i><span id="fmsg" style="float:right;font-size:75%">${flashMessage}</span></i>
```

Comment:
There are pros and cons to doing it inline vs in css, but since nearly all of our styles come from classes in external css and not inline. It would fit the code base more to do it in `plugin.scss`